articles/active-directory/manage-apps/disable-user-sign-in-portal.md

```diff
@@ -33,6 +33,10 @@ It's easy to disable an enterprise application so no users can sign in to it in
 1. On the ***appname*** - **Properties** pane, select **No** for **Enabled for users to sign-in?**.
 1. Select the **Save** command.
 
+## Disable unlisted Enterprise apps
+
+To disable apps that uses OAuth2 implicit flow, the tenant admin must manually create and disable the ServicePrincipal in their tenant using the [AzureAD PowerShell cmdlet](https://docs.microsoft.com/en-us/powershell/module/azuread/New-AzureADServicePrincipal?view=azureadps-2.0). For e.g. if the tenant admin wants to disable a public client app (SPA, mobile, etc.) that calls Microsoft Graph, then the Service Principal would need to be created and disabled using PowerShell.
```

Comment:
The audience of this article (admins) is unlikely to know (or care) what the "OAuth2 implicit flow" is. Also, the scenario in question is not specific to apps using the Implicit flow, it applies to any app using a public client flow which has been pre-authorized for the permissions it is requesting. (For example, this would also apply for Azure AD PowerShell, Azure CLI, etc., which are both natice client apps apps use Authorization Code flow as public clients, and do not typically get a ServicePrincipal object in a customer's tenant.)

In practice, this scenario only ends up applying to Microsoft-published apps, since those are the only ones which can be pre-authorized for things on Microsoft Graph or Azure AD Graph (e.g. `User.Read` or `openid` scopes). So, I suggest re-writing this to say: "Some Microsoft apps will not show up in the Enterprise apps list. If you wish to disable these, you can create a service principal for the app in your tenant using Azure AD PowerShell, and the proceed to disable it."

Reply:
updated the description.